vs.language.puppet/puppetDef.js

```diff
@@ -277,16 +275,15 @@ define(["require", "exports"], function (require, exports) {
                 [/.*/, 'string.heredoc'],
             ],
             // interpolated sequence
-            interpolated: [
-                [/\$\w*/, 'global.constant', '@pop'],
-                [/@\w*/, 'namespace.class.identifier', '@pop'],
-                [/@@\w*/, 'namespace.instance.identifier', '@pop'],
-                [/[{]/, { token: 'string.escape.curly', bracket: '@open', switchTo: '@interpolated_compound' }],
+            interpolated: [            
+                [/[{]/, { token: 'annotation', bracket: '@open', switchTo: '@interpolated_compound' }],
+                [/\w*/, 'annotation', '@pop'],
                 ['', '', '@pop'],
             ],
             // any code 
             interpolated_compound: [
-                [/[}]/, { token: 'string.escape.curly', bracket: '@close', next: '@pop' }],
+                [/[}]/, { token: 'annotation', bracket: '@close', next: '@pop' }],
+                [/\w*/,'keyword'],

```

Comment:
stickout in string more boldly, surrounded by annotation color for ${variable} string interpolation